Python/PR_Analyzer/main.py

```diff
@@ -0,0 +1,381 @@
+#Necessary Packages
+import datetime
+import json
+import re
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+from jupyterthemes import jtplot
+from matplotlib import pyplot as plt
+from sklearn.cluster import KMeans
+
+jtplot.style('monokai','notebook',ticks=True)
+
+
+#---------------------------------------------------------------------------------------------------------#
+
+"""
+This Section contains code block to fetch all the Pull Requests along with title.
+The result is zipped to a Pandas DataFrame.
+"""
+#Code block to fetch the all Pull Requests along with their links.
+prs=[]
+no=[]
+info=[]
+pr_title=[]
+for x in range(1,30):
+    r=requests.get(url="https://github.com/HarshCasper/Rotten-Scripts/pulls?page="+str(x)+"&q=is%3Apr")
+    soup=BeautifulSoup(r.content,'html.parser')
+    div_tags=soup.findAll('div',class_="Box-row Box-row--focus-gray p-0 mt-0 js-navigation-item js-issue-row")
+    base_url="https://github.com/"
+    for _,i in enumerate(div_tags):
+        for a in i.findAll('a'):
+            newUrl=base_url+a["href"]
+        no.append(_)
+        info.append(i.text.strip())
+        prs.append(newUrl)
+for item in info:
+    pr_title.append(re.sub("\n","",item))
+
+#Converting the fetched information into a Python DataFrame
+data=list(zip(no,pr_title,prs))
+df=pd.DataFrame(data,columns=["No","Info","PR Url"])
+for i in range(len(df)):
+    df['No'][i]=i
+
+print(df)
```

Comment:
Remove all print() statements that we don't generally want.

Reply:
Yep, removed them as well.